packages/cli/test/integration/generators/app.integration.js

```diff
@@ -56,7 +56,21 @@ describe('app-generator specific files', () => {
       'src/controllers/ping.controller.ts',
       /\'\@loopback\/openapi\-v3\'/,
     );
+  });
+});
 
-    assert.file;
+describe('app-generator with --applicationName', () => {
+  before(() => {
+    return helpers
+      .run(generator)
+      .withOptions({applicationName: 'MyApp'})
+      .withPrompts(props);
```

Comment:
Shall we also add the naming convention in doc?
- For the `applicationName` provided from prompt, `my-app` generates a class called `MyAppApplication`, which appends `Application` after the camel case `my-app`
- For the `applicationName` provided from option, we use exactly the input as class name.

Reply:
The prompt derives `applicationName` from the project name by appending `Application`. But the answer is honored if a user decides to override it. There is no difference between two cases.